### DISCUSSION TITLE
Inconsistent rendering of opacity in <G> element on Android and iOS

### DISCUSSION THREAD

**Discussion by uier** [software-mansion/react-native-svg#2035] [Q&A] [answered]
When using the <G> element with an opacity attribute, the rendering of nested elements inside the <G> element is inconsistent between Android and iOS platforms.

**To Reproduce**
Here's a minimal code snippet that reproduces the issue:

```jsx
import { Svg, G, Circle, Rect } from "react-native-svg";

export function Overlap() {
    return (
        <Svg>
            <G opacity={0.5} translateX={100} translateY={200}>
                <Circle r="40" fill="black" />
                <Rect width="80" height="60" fill="green" />
            </G>
        </Svg>
    )
}
```

The left one is on Android emulator (Pixel_6_Pro_API_33), the right one is on iOS simulator (iPhone 14 Pro)

<div style="display: flex">
  <img width="345" alt="Screenshot 2023-04-15 at 1 36 31 PM" src="https://user-images.githubusercontent.com/16915098/232185364-2f8f7eea-fce6-4cf3-b964-ec7307b232dd.png">
  <img width="345" alt="Screenshot 2023-04-15 at 1 26 32 PM" src="https://user-images.githubusercontent.com/16915098/232185350-db28ce16-04e1-4686-9dc0-32c5fe8d2e08.png">
</div>

How can I achieve the iOS result (the contents of the group to first be composited and then the result to be lowered in opacity) on Android?

Thank you!

**Comment by jakex7** [comment, marked as answer]
Hey!  
This issue has already been fixed in [this PR](https://github.com/software-mansion/react-native-svg/pull/2417. Please try updating to the latest version, and feel free to ping me if you encounter any further problems!
